packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -158,15 +170,106 @@ class FloatingActionButton extends StatelessWidget {
        assert(isExtended != null),
        assert(autofocus != null),
        _sizeConstraints = mini ? _kMiniSizeConstraints : _kSizeConstraints,
+       _floatingActionButtonType = mini ? _FloatingActionButtonType.small : _FloatingActionButtonType.regular,
+       _label = null,
+       extendedIconLabelSpacing = null,
+       super(key: key);
+
+  /// Creates a small circular floating action button.
+  ///
```

Comment:
All of the factory constructors should specify what features of the default FAB they are overriding.

Reply:
Done